docs/S3.md

```diff
@@ -6,6 +6,9 @@ If using the default endpoint, you **must** configure the region using the `SCCA
 
 If your endpoint requires HTTPS/TLS, set `SCCACHE_S3_USE_SSL=true`. If you don't need a secure network layer, HTTP (`SCCACHE_S3_USE_SSL=false`) might be better for performance.
 
+Enable server side encryption with s3 managed key (SSE-S3), set `SCCACHE_S3_SERVER_SIDE_ENCRYPTION=true` .  
+More details about encryption [here](https://opendal.apache.org/docs/services/s3/#server-side-encryption) and documentation [here](https://docs.rs/opendal/latest/opendal/services/struct.S3.html#method.server_side_encryption_with_s3_key).
```

Comment:
please add the link on "encryption" and "documentation"
"here" is pretty bad for accessibility / screen reader. thanks